src/main.rs

```diff
@@ -289,6 +304,9 @@ impl SeatHandler for State {
             .unwrap();
         let keyboard = self.keyboards.swap_remove(i);
         keyboard.destroy(conn);
+        if self.keyboard_shortcuts_inhibit_manager.is_some() {
+            self.keyboard_shortcuts_inhibitors.swap_remove(i);
+        }
```

Comment:
1. These lines should be moved to `SeatHandler::seat_removed`.
2. The inhibitor should be [destroyed](https://docs.rs/wayrs-protocols/latest/wayrs_protocols/keyboard_shortcuts_inhibit_unstable_v1/zwp_keyboard_shortcuts_inhibitor_v1/struct.ZwpKeyboardShortcutsInhibitorV1.html#method.destroy).

Reply:
Done.